kb/disorders/Frontotemporal_Dementia.yaml

```diff
@@ -0,0 +1,184 @@
+name: Frontotemporal Dementia
+description: >
+  Frontotemporal Dementia (FTD) is a group of neurodegenerative disorders characterized
+  by progressive deterioration of the frontal and temporal lobes of the brain. It presents
+  with prominent changes in personality, behavior, and/or language, typically with onset
+  before age 65. FTD is associated with abnormal accumulation of tau or TDP-43 proteins.
+category: Complex
+disease_term:
+  preferred_term: frontotemporal dementia
+  term:
+    id: MONDO:0017276
+    label: frontotemporal dementia
+parents:
+- Dementia
+- Neurodegenerative Disease
+pathophysiology:
+- name: TDP-43 Proteinopathy
```

Comment:
agent: consider adding some kind of GO term plus qualifier to indicate the process/function that is aberrant. Or in this case we might need a new ontology for pathophysiology. Consider also indicating the target e.g. uniprot. This may require extending the data model